foundation/foundation-mda/src/main/resources/templates/deployment/spark-operator/spark-application-base-values.yaml.vm

```diff
@@ -69,7 +69,7 @@ sparkApp:
     driver:
       cores: 1
       coreLimit: "1200m"
```

Comment:
S (Substantive): maybe I'm off, but my impression is that the driver doesn't need nearly as much memory in general as the nodes?  

When we work with memory settings, they quickly add up across a cluster.  So I think we want the defaults to be as small as we can reasonably make them, but no smaller.  How did we arrive at these values?

Reply:
I think Ryan is commenting on our memory changes and just happened to attach it to the coreLimit line.  I think it's a very fair point.  We still have seen issues with driver memory being too low at 512m, but It might make sense to halve the memory assignment here to 2g to match what we've seen as the bare minimum that works on all machines for what amounts to no-op pipelines.

FWIW, I think we tried to do a survey of what most projects were configuring here to arrive at the 4g, but I'm not sure if we were careful to distinguish between driver/executor, or if the projects we looked at cared to distinguish.

Reply:
We've set the default value for base-values.yaml to 4g as most downstream projects use 4g. To buffer this, we set 2g for dev-values.yaml for local. If it's still too much allocation, we can remove the 4g setting.